test/e2e-test-application/src/assets/helloWorldWC.js

```diff
@@ -29,6 +29,12 @@ export default class extends HTMLElement {
           text: 'Hello from uxManager in Web Component, Language:' + this.LuigiClient.getCurrentLocale(),
           type: 'info'
         });
+
+        this.LuigiClient.uxManager().showAlert({
```

Comment:
Could you add a test for this?
Please take a look at https://github.com/SAP/luigi/blob/master/test/e2e-test-application/e2e/tests/1-angular/luigi-client-link-manager-features.spec.js#L117
E.g. a new test where a feature toggle is set and check in the web components view if the alert has the correct feature toggle list.

Reply:
Moved it to a separate test case as we discussed in the daily :).